compiler/llvm.go

```diff
@@ -102,6 +101,70 @@ func (c *compilerContext) createObjectLayout(t llvm.Type, pos token.Pos) llvm.Va
 		layout := (uint64(1) << 1) | 1
 		return llvm.ConstIntToPtr(llvm.ConstInt(c.uintptrType, layout, false), c.i8ptrType)
 	}
+	return c.createObjectLayout1(pointerSize, objectSizeBytes, bitmap, pos)
```

Comment:
My version passed a pointerSize. But with the change to line 163, this should also be the alignment. Maybe better to pass both or pass neither and just recompute them in createObjectLayout1.